src/fideslang/models.py

```diff
@@ -839,20 +929,49 @@ class PrivacyDeclaration(BaseModel):
     ingress: Optional[List[FidesKey]] = Field(
         description="The resources from which data is received. Any `fides_key`s included in this list reference `DataFlow` entries in the `ingress` array of any `System` resources to which this `PrivacyDeclaration` is applied."
     )
+    features: List[str] = Field(
+        default=[], description="The features of processing personal data."
```

Comment:
Default values like this are dangerous in Python and can lead to some nasty unintended side-effects (been there, felt that pain!)

```suggestion
        default_factory=list, description="The features of processing personal data."
```

Reply:
My understanding was that the using a default=[] or `default_factory`=list results in the same behavior though, given that Pydantic deepcopies the default value for each new instance  https://docs.pydantic.dev/latest/usage/models/#fields-with-non-hashable-default-values

Can you clarify how this will make a difference?

Reply:
You're totally right, I was mistaken! It looks like Pydantic fixes Python's typical unexpected behavior here 🙂 

TIL!